examples/qml_features/rust/src/externcxxqt.rs

```diff
@@ -22,6 +22,11 @@ pub mod ffi {
         #[qsignal]
         fn triggered(self: Pin<&mut Self>);
 
+        /// const signal that is emitted when trigger is fired
+        #[qsignal]
+        #[rust_name = "triggered_const_signal"]
+        fn triggeredConstSignal(&self);
```

Comment:
you should add a call to this here

https://github.com/KDAB/cxx-qt/blob/main/examples/qml_features/cpp/external_qobject.cpp#L18

And copy the connection here 

https://github.com/KDAB/cxx-qt/blob/main/examples/qml_features/rust/src/externcxxqt.rs#L69

And then you can add a signal spy for it and check that they are triggered here so it's tested :-) 

https://github.com/KDAB/cxx-qt/blob/main/examples/qml_features/tests/tst_externcxxqt.qml#L41